AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
Why do GPU and CPU variants/duplicates need to exist? Is the CPU variant registered anywhere?

Reply:
The linked smoke test doc had requirements for both cpu + gpu. The cpu tests should be ran during AR submission

Reply:
Are the CPU tests run during submission? The CMakeLists.txt appears to only reference GPU.

Reply:
GH won't let me comment directly on the line, but there appears to be a different registration block that is registering everything in the directory to run in the Smoke suite, which will probably cause the GPU test to also run in the smoke suite.

Reply:
The smoke test registration has the "SUITE_smoke" pytest mark requirment. The gpu test does not have that mark and is registered separately so it won't run in the same suite

Reply:
And yes cpu tests are ran during submission. Here is an sample AR run for this PR and you can see there is a cpu profile job https://jenkins.build.o3de.org/blue/organizations/jenkins/O3DE/detail/PR-1997/4/pipeline/528